checks.d/etcd.py

```diff
@@ -68,6 +69,18 @@ def check(self, instance):
         # Load values from the instance config
         url = instance['url']
         instance_tags = instance.get('tags', [])
+
+        # Load the ssl configuration
+        ssl_params = {
+            'ssl_keyfile': instance.get('ssl_keyfile'),
+            'ssl_certfile': instance.get('ssl_certfile'),
+            'ssl_cert_validation': _is_affirmative(instance.get('ssl_cert_validation', True)),
```

Comment:
By default ssl_cert_validation is enabled: it could be a breaking change for people using the current check with an invalid cert.